cmd/utils/flags.go

```diff
@@ -1186,6 +1186,7 @@ func SetP2PConfig(ctx *cli.Context, cfg *p2p.Config) {
 	} else if forceV5Discovery {
 		cfg.DiscoveryV5 = true
 	}
+	cfg.DnsDiscovery = lightClient && !ctx.GlobalBool(NoDiscoverFlag.Name)
```

Comment:
Why to make the `cfg.DnsDiscovery` LES specific?

Reply:
Can we change it to `cfg.DnsDiscovery = !ctx.GlobalBool(NoDiscoverFlag.Name))` ? In this way the field is both suitable for ETH and LES